package.json

```diff
@@ -35,35 +35,30 @@
   "author": "Peter Staev <peter@tangrasoft.com>",
   "contributors": [
     "Dick Smith (https://github.com/DickSmith)"
-  ],  
+  ],
   "license": "Apache-2.0",
-  
   "peerDependencies": {
-     "tns-core-modules": "^3.0.0"
+    "tns-core-modules": "^3.1.0 || >3.3.0-"
   },
-  
   "devDependencies": {
-    "@angular/common": "~4.0.3",
-    "@angular/compiler": "~4.0.3",
-    "@angular/compiler-cli": "~4.0.3",
-    "@angular/core": "~4.0.3",
-    "@angular/forms": "~4.0.3",
-    "@angular/http": "~4.0.3",
-    "@angular/platform-browser": "~4.0.3",
-    "@angular/router": "~4.0.3",
-    "rxjs":"~5.0.1",
-    "zone.js": "~0.8.9",
-    "nativescript-angular": "3.0.0",
-
-    "typescript": "~2.2.2",
-    "tslint": "^4.5.1",
-    "tns-core-modules": "3.0.1",
-    "tns-platform-declarations": "3.0.1",
-
+    "@angular/common": "~4.4.1",
+    "@angular/compiler": "~4.4.1",
+    "@angular/compiler-cli": "~4.4.1",
+    "@angular/core": "~4.4.1",
+    "@angular/forms": "~4.4.1",
+    "@angular/http": "~4.4.1",
+    "@angular/platform-browser": "~4.4.1",
+    "@angular/router": "~4.4.1",
     "grunt": "1.0.1",
-    "grunt-contrib-copy": "1.0.0",
     "grunt-contrib-clean": "1.0.0",
-    "grunt-exec": "2.0.0"
+    "grunt-contrib-copy": "1.0.0",
+    "grunt-exec": "2.0.0",
+    "nativescript-angular": "4.4.0",
+    "rxjs": "~5.0.1",
+    "tns-core-modules": "next",
```

Comment:
Similar thing as above. I prefer this to remain at the least supported version so unless it needs anything from newer modules this should remain at 3.0.1.